src/api/event.ts

```diff
@@ -15,21 +15,63 @@ import { AptosConfig } from "./aptosConfig";
 import { waitForIndexerOnVersion } from "./utils";
 
 /**
- * A class to query all `Event` Aptos related queries
+ * A class to query all `Event` Aptos related queries.
  */
 export class Event {
+
+  /**
+   * Initializes a new instance of the Aptos client with the provided configuration.
```

Comment:
ohh it looks like you added it to any API class - not completely sure why

Reply:
In order for the comments to show up in TS Doc, we have to add comments to the API classes since they're used as mixins to define the functions on `Aptos`. These comments are what actually explain each method :)